omod/src/main/webapp/admin/users/userForm.jsp

```diff
@@ -103,9 +103,42 @@
 			</tr>
 		</table>
 	</fieldset>	
-	
 	<br/>
 	
+    <fieldset>
+        <legend>Provider Account</legend>
```

Comment:
Have a look at the ticket comments and images. It is explained there. Basically, there are 2 possibilities. If the user is new, we display a check box to add him to provider table 

Else, we display a table of all the providers the person associated with the user is.